### PR TITLE
Add /firefox/whatsnew_67.0.5.lang

### PR DESCRIPTION
**Pull request by amychurchwell**
Bedrock issue: https://github.com/mozilla/bedrock/issues/7128
Bedrock PR: https://github.com/mozilla/bedrock/pull/7206
Langchecker PR: https://github.com/mozilla-l10n/langchecker/pull/883

### CODE REVIEW

**Comment by amychurchwell** [comment]
Hi @peiying2 the ask is for en, de, and fr locales as priority. Someone in the marketing dept will provide the de translation once this is Pontoon and someone from the Mother Tongue service will provide fr. All other locales are requested, but not critical for launch. (June 4, 2019)

**Comment by flodolo** [comment]
Per previous email conversations, this should *not* be exposed to any other locales. The page is only relevant once, and it's not going to be used for anything but the target languages.

Target should be fr, de, en-GB, en-CA. We'll need to make sure en-GB is activated even if incomplete.

**Comment by amychurchwell** [comment]
@flodolo yes, you are right. Thanks, Flod. 🙂

File: en-US/firefox/whatsnew_67.0.5.lang [hunk 1]
```diff
@@ -0,0 +1,92 @@
+
```
Comment (peiying2) [inline en-US/firefox/whatsnew_67.0.5.lang]: Need to include testing info. Reference this for examples: https://github.com/mozilla-l10n/www.mozilla.org/blob/master/en-US/firefox/new/trailhead.lang

**Comment by amychurchwell** [comment]
@peiying2 the page won't be on demo until 5/21. Added bedrock issue # to description (https://github.com/mozilla/bedrock/issues/7128)

File: en-US/firefox/whatsnew_67.0.5.lang (hunk 1 above)
Comment (amychurchwell) [reply en-US/firefox/whatsnew_67.0.5.lang]: I can include what the demo link will be when it gets pushed up to demo on 5/21. However, this wont be exposed to the l10n community. fr and de strings will be provided once this is in Pontoon.

**Comment by amychurchwell** [comment]
@peiying2 thank you for the helpful feedback! Added notes about the product names and mentioned that the demo url is TBD at the top.

File: en-US/firefox/whatsnew_67.0.5.lang [hunk 2]
```diff
@@ -0,0 +1,93 @@
+## NOTE: Demo page TBD
+
```
Comment (peiying2) [inline en-US/firefox/whatsnew_67.0.5.lang]: Need double line space between entries. Check the entire doc for single-line spaces.

Comment (amychurchwell) [reply en-US/firefox/whatsnew_67.0.5.lang]: Ok, spacing should be fixed now.

**Comment by amychurchwell** [comment]
@alexgibson Peiying & I were struggling today to get this into Pontoon for Anika. Would you be able to pick this up in your morning tomorrow? I wasn't able to get `lang_update` to work for me on my machine & Peiying was not able to pick it up where I left it.

**Comment by flodolo** [comment]
> I wasn't able to get `lang_update` to work for me on my machine

Can we make sure to figure it out? I'll take care of updating this.

**Comment by flodolo** [comment]
Push to prod: https://github.com/mozilla-l10n/bedrock-l10n/commit/0b82d249fed7352c27677cb2626359ae5ff9fb8a

**Comment by amychurchwell** [comment]
@flodolo Thank you, Flod! Will get that sorted.

**Pull request merged**